tests/posix/common/src/mutex.c

```diff
@@ -28,7 +28,7 @@ void *normal_mutex_entry(void *p1)
 		if (rc == 0) {
 			break;
 		}
-		k_sleep(SLEEP_MS);
+		k_sleep(K_MSEC(SLEEP_MS));
```

Comment:
You can use `k_msleep()` instead

Reply:
Done

Reply:
>> You can use k_msleep() instead

> Done

No, wait, k_sleep has prototype k_sleep(k_timeout_t timeout), but k_msleep - k_msleep(s32_t ms)? But that makes no sense!

Reply:
k_msleep() and k_usleep() are simple wrappers that take integers in specific  units; they work like APIs in other systems.  There was desire last summer that k_sleep() take a k_timeout_t to allow it to use multiprecision and absolute arguments (so people who wanted that wouldn't have to use a k_timer, basically).

The suggestion was to change that code to `k_msleep(SLEEP_MS);`